Source/WTF/wtf/FastMalloc.h

```diff
@@ -31,15 +31,35 @@ namespace WTF {
 // allocation is handled by libpas or bmalloc, depending on which if any of these memory allocators is available.
 //
 // TLDR; Here is a quick guidance.
-//   1. If your class / struct is derived from a base class which uses WTF_MAKE_TZONE_ALLOCATED / WTF_MAKE_TZONE_ALLOCATED_EXPORT,
-//      you must use WTF_MAKE_TZONE_ALLOCATED / WTF_MAKE_TZONE_ALLOCATED_EXPORT.
-//   2. If your class / struct is derived from a base class which uses WTF_MAKE_TZONE_OR_ISO_ALLOCATED / WTF_MAKE_TZONE_OR_ISO_ALLOCATED_EXPORT,
-//      you must use WTF_MAKE_TZONE_OR_ISO_ALLOCATED / WTF_MAKE_TZONE_OR_ISO_ALLOCATED_EXPORT.
-//   3. If your class / struct is a DOM object, use WTF_MAKE_TZONE_OR_ISO_ALLOCATED.
-//   4. If your class / struct is particularly memory consuming and if you think tracking footprint of your class is helpful for memory-reduction work,
-//      use WTF_MAKE_FAST_ALLOCATED_WITH_HEAP_IDENTIFIER / WTF_MAKE_FAST_ALLOCATED_WITH_HEAP_IDENTIFIER.
-//   5. For classes / structs that are fixed sized, use WTF_MAKE_TZONE_ALLOCATED / WTF_MAKE_TZONE_ALLOCATED_EXPORT.
+//   1. If your class / struct /template is derived from a base class which uses:
+//          WTF_MAKE_TZONE_ALLOCATED
+//          WTF_MAKE_TZONE_ALLOCATED_EXPORT
+//          WTF_MAKE_STRUCT_TZONE_ALLOCATED
+//          WTF_MAKE_TZONE_ALLOCATED_TEMPLATE
+//          WTF_MAKE_TZONE_ALLOCATED_TEMPLATE_EXPORT
+//          WTF_MAKE_COMPACT_TZONE_ALLOCATED
+//          WTF_MAKE_COMPACT_TZONE_ALLOCATED_EXPORT
+//      then you must use the same TZONE_ALLOCATED annotation as well.
```

Comment:
This comment seems slightly inaccurate? If I had `class Foo : public Bar<Baz>`, `Bar` would be `WTF_MAKE_TZONE_ALLOCATED_TEMPLATE` but my class isn't templated so I think I'd use `WTF_MAKE_TZONE_ALLOCATED`. Is that right?

Reply:
Yes, you would just use WTF_MAKE_TZONE_ALLOCATED or WTF_MAKE_TZONE_OR_ISO_ALLOCATED.

Reply:
I see what you mean.  I'll update it.